S100-Permit-Service-Open-Api-Sepc-v0.1.yaml

```
@@ -0,0 +1,162 @@
+        "204":
+          description: No Content - There are no S100 permits for the licence (No UPNs or Holdings found for the requested licence from S100 Shop Facade).
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "400":
+          description: Bad request - could be missing or invalid licenceId, it must be an integer and greater than zero.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: licenceId
+                  description: Invalid LicenceId
+            application/zip:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+        "401":
+          description: Unauthorised - either you have not provided valid token, or your token is not recognised.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content: {}
+        "403":
+          description: Forbidden - you have no permission to use this API.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "404":
+          description: Licence not found (licence not found in response received from Shop Facade UPN Service or Shop Facade Holding Service).
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: licenceId
+                  description: Licence not found.
+            application/zip:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+        "429":
+          description: Too Many Requests.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "500":
+          description: Internal Server Error (reasons could be dependent services returns status code 429/401/403/503/500 or error in processing UPNs / permit key or error in generating permit or error in generating zip file).
```

```suggestion
          description: Internal Server Error.
```
I suggest internal wokings of API shouldn't be included in API spec.

---

Agree, Updated.